sw/host/cryptotest/testvectors/parsers/nist_cavp_hmac_parser.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Parser for converting NIST CAVP Digital Signatures test vectors to JSON.
+
+"""
+# TODO: Add more detailed docstring
```

Comment:
Is this TODO still relevant?

Reply:
Nope, that was copied from the AES parser and I didn't catch it.